app/views/whats_new/_type.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+  type_class = case type.downcase
+                when "new" then "govuk-tag--green"
+                when "improvement" then "govuk-tag--blue"
+                when "change" then "govuk-tag--red"
+                when "fix" then "govuk-tag--yellow"
+               end
+%>
+
+<strong class="govuk-tag <%= type_class %>">
+  <%= type %>
+</strong>
```

Comment:
I don't really think we'll need a type system for changes, I guess this is just from copy+pasting from Whitehall. But I imagine in total we'll only have at most 5-6 updates and I think they'd all be considered improvements.